packages/document/doc-tools-doc/modern.config.ts

```diff
@@ -50,6 +50,11 @@ export default defineConfig({
           label: 'English',
         },
       ],
+      editLink: {
+        repoUrl: 'https://github.com/test/blog',
+        text: 'Edit this page on GitHub',
+        pattern: '',
```

Comment:
The `pattern` param is not neccessary because we can join  doc repo base url and relative page path and take the result as edit link url.